src/scss/main.scss

```diff
@@ -1,99 +1,95 @@
 // Make sure the charset is set appropriately
 @charset "UTF-8";
-
 // Foundation Components
-@import
-"../../node_modules/foundation-apps/scss/foundation";
-
+@import "../../node_modules/foundation-apps/scss/foundation";
 @import "../../node_modules/angular-toastr/dist/angular-toastr";
-
-
 body {
-	margin: 0;
+    margin: 0;
 }
 
-.franklin--login--area{
-	padding-top: 15%;
+.franklin--login--area {
+    padding-top: 15%;
 }
 
-.franklin--login{
-	height: 100vh;
-	background: #1e5799; /* Old browsers */
-	background: -moz-radial-gradient(center, ellipse cover, #1e5799 0%, #89b56a 0%, #6fb240 40%, #5d8f3c 100%); /* FF3.6-15 */
-	background: -webkit-radial-gradient(center, ellipse cover, #1e5799 0%,#89b56a 0%,#6fb240 40%,#5d8f3c 100%); /* Chrome10-25,Safari5.1-6 */
-	background: radial-gradient(ellipse at center, #1e5799 0%,#89b56a 0%,#6fb240 40%,#5d8f3c 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
-	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#1e5799', endColorstr='#5d8f3c',GradientType=1 ); /* IE6-9 fallback on horizontal gradient */
-	width: 100%;
-	color: #CCC;
-	text-align: center;
+.franklin--login {
+    height: 100vh;
+    background: #1e5799;
+    /* Old browsers */
+    background: -moz-radial-gradient(center, ellipse cover, #1e5799 0%, #89b56a 0%, #6fb240 40%, #5d8f3c 100%);
+    /* FF3.6-15 */
+    background: -webkit-radial-gradient(center, ellipse cover, #1e5799 0%, #89b56a 0%, #6fb240 40%, #5d8f3c 100%);
+    /* Chrome10-25,Safari5.1-6 */
+    background: radial-gradient(ellipse at center, #1e5799 0%, #89b56a 0%, #6fb240 40%, #5d8f3c 100%);
+    /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+    filter: progid: DXImageTransform.Microsoft.gradient( startColorstr='#1e5799', endColorstr='#5d8f3c', GradientType=1);
+    /* IE6-9 fallback on horizontal gradient */
+    width: 100%;
+    color: #CCC;
+    text-align: center;
 }
 
-.franklin--login--button a{
-	background-color: #FFF;
-	border-color: #FFF;
-	border-style: none;
-	color: #5D8F3C;
-	font-size: 20px;
-	border: 2px solid #FFF;
-	font-family: 'Century Gothic', CenturyGothic, AppleGothic, sans-serif;
-	border-radius: 30px;
+.franklin--login--button a {
+    background-color: #FFF;
+    border-color: #FFF;
+    border-style: none;
+    color: #5D8F3C;
+    font-size: 20px;
+    border: 2px solid #FFF;
+    font-family: 'Century Gothic', CenturyGothic, AppleGothic, sans-serif;
+    border-radius: 30px;
 }
 
-.franklin--login--button a:hover{
-	cursor: pointer;
-	background-color: #5D8F3C;
-	color: #FFF;
+.franklin--login--button a:hover {
```

Comment:
We're violating BEM standards here, we shouldn't have two sets of double quotes. This applies to several other rules in this file.